image_width: 1440
image_height: 1080
camera_name: hk_camera_left
camera_matrix:
  rows: 3
  cols: 3
  data: [ 1793.45495,    0.     ,  761.63528,
          0.     , 1791.83362,  551.58477,
          0.     ,    0.     ,    1. ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [ -0.079407, 0.113988, -0.000268, -0.000251, 0.000000 ]
rectification_matrix:
  rows: 3
  cols: 3
  data: [ 1., 0., 0.,
          0., 1., 0.,
          0., 0., 1. ]
projection_matrix:
  rows: 3
  cols: 4
  data: [ 1774.34851,    0.     ,  761.46581,    0.     ,
          0.     , 1778.79749,  551.0661 ,    0.     ,
          0.     ,    0.     ,    1.     ,    0. ]
extrinsic:
  rows: 3
  cols: 4
  data: [ 0.152768,  -0.988227, -0.00832467 , 0.0501988,
          0.00847187 , 0.00973279  ,-0.999917 , 0.0369495,
          0.988226  ,0.152685 , 0.00985899 , 0.0125274 ]
extrinsic2:
  rows: 3
  cols: 4
  data: [ 0.167909,  -0.985784,  -0.00612153,  0.0590264,
          -0.0331205,  0.000564986,  -0.999451,  0.0405572,
          0.985246,  0.168019,  -0.0325548,  -0.060554 ]
world_to_cam:
  rows: 3
  cols: 4
  data: [0.9902065104383774, -0.1381551706427959, 0.02010511154187618, -1.80592971081812,
    -0.007843078247378649, 0.08873296532120606, 0.9960245714784907, 1.264309490685013,
    -0.1393899308022529, -0.9864277011975918, 0.08678039813792962, 30.89981232167481]

#3D_points:  # Must be float (Ref: https://github.com/daniilidis-group/msckf_mono/issues/28)
#
#  #left_camera
#  p1: [ 0.927998, 0.926115, -1.531]
#  p2: [2.891553, 11.815893, -1.447749]
#  p3: [ 8.311249, 14.811249, -2.386 ]
#  p4: [ 1.61, 16.712008, -0.735 ]
#  p5: [6.197724, 19.666201, -0.614995]
#  p6: [ 6.857724, 19.666201, -0.614995 ]

  #right_camera
#  p1: [ 8.311249, 14.811249, -2.386 ]
#  p2: [12.992708, 17.371113, -1.578351]
#  p3: [ 12.78, 19.031606, -0.945]
#  p4: [6.197724, 19.666201, -0.614995]
#  p5: [ 6.857724, 19.666201, -0.614995 ]
#  p6: [ 10.07, 23.224486, -1.4]

#  p1: [2.891553, 11.815893, -1.447749]
#  p2: [13.048568, 17.43886, -1.580651]
#  p3: [12.992708, 17.371113, -1.578351]
#  p4: [6.197724, 19.666201, -0.614995]
#  p5: [ 6.857724, 19.666201, -0.614995 ]
#  p6: [ 6.857724, 19.666201, -0.614995 ]
#  p7: [ 8.311249, 14.811249, -2.386 ]
#  p8: [ 1.61, 16.712008, -0.735 ]
#  p9: [ 0.927998, 0.926115, -1.531]
#  p10: [ 12.78, 19.031606, -0.945]
#  p11: [ 5.87, 23.224486, -1.4 ]
#  p12: [ 10.07, 23.224486, -1.4]

